src/main/java/page_objects/ContactPage.java

```diff
@@ -0,0 +1,13 @@
+package page_objects;
+
+import helpers.BrowserHelper;
+import org.openqa.selenium.By;
+
+public class ContactPage extends BasePage {
+    private final By lnkEmail = By.linkText("thanh.viet.le@logigear.com");
```

Comment:
nooooo, you want to check for `mailto:thanh.viet.le@logigear.com`, but you capture an element with that value inside, find another way.

Reply:
I've captured with new CSS Selector. Please check and review it. Thank you!